client/my-sites/stats/stats-module-devices/stats-module-upgrade-overlay.tsx

```diff
@@ -64,10 +64,10 @@ const StatsModuleUpgradeOverlay: React.FC< StatsModuleUpgradeOverlayProps > = (
 				<StatsCardUpsellJetpack
 					className="stats-module__upsell"
 					siteId={ siteId }
-					statType={ STATS_TYPE_DEVICE_STATS }
+					statType={ STATS_FEATURES_DEVICES_EVENT_NAME }
```

Comment:
Maybe we could just change the value of `STATS_TYPE_DEVICE_STATS` to `stats_devices_module`? It feels like more natural to me 🤔

Reply:
Ok, I updated the name.